ui/src/app/app.module.ts

```diff
@@ -30,7 +30,6 @@ import {AppsModule} from './apps/apps.module';
 import {UrlUtilities} from './url-utilities.service';
 import {TaskService} from './shared/api/task.service';
 import {JobService} from './shared/api/job.service';
-import {DevModule} from './dev/dev.module';
```

Comment:
What is the impact of removing this?

Reply:
The impact is that you cannot navigate to `DashboardComponent` now via `/dev`. I suspect that this is intentional as `DevModule` seems experimental... There is no navigation to it via UI...  only direct url with `/dev` in the path. The dashboard seem to require a bit of CSS work (i.e. paddings between buttons). Likely intentional...